### ISSUE TITLE
Configmap for OBC being placed in kube-system?

### ISSUE THREAD

**Issue by amx-jshriner** [yard-turkey/aws-s3-provisioner#82]
I am building a helmchart using this operator. According to the documentation, the configmap referenced by my pod(s) should be available in the same namespace as the OBC, but when I look, it's actually created in kube-system and giving itself the namespace's name and versioning it.

Example:
![image](https://user-images.githubusercontent.com/58048071/73282105-06640a80-41bf-11ea-9d76-db980d211f17.png)

Any ideas what I might be doing wrong?

**Comment by amx-jshriner** [comment]
Just updating the thread with the actual configmap yaml, as it may help my case...

`apiVersion: v1
data:
  release: H4sIAAAAAAAC/+xZzW8jSRXXJjtiVcNKqwiBtNrDmxixYG234yQ7GllaoYzj2fnMlyeJsqNoVN39bNe4uqq3qtqJZ7WIC38Fgn8AIf4BLnDgxA2BOHCBE1fEgSuqqm5/xU7CMhyA+NDqrnr13qv3fu+jyuRreiMwqM3KOll6Z3nl6+/87Oe//tut937yi9/9dPn9qa/qe4+ENpRziGWacTT4/l/vkO+POKzeWgvr4Vr127sZKmqkAiMhlbkw0N6AKI/7aDRQDXtHTX2wNKi/Wq6Hayt/XCbfMphmnBrUtVhhogO9EQ5pyld+uUwzdoRKMykaMKiTPhNJA9oYKzQkRUMTamiDAAiaYgP0RuDlBPJMoILKi/ppMakzGnsKqytUXqyfEjPMsAG7Gf08R1Jy2jpuv9xqNlvt9ssnrZOXj7YbsH9oHu8/f9w8fJ20ntWP9g9b9Z39tXjt4LD7CVRebJwWy9qt5kHr+cTqBkTr9az9ae8kEsng+NXjJwcPP1t/srbz5LDeH+4eH7QP6g+aJ8eZ+exhf3DUerARr+9sHu1/YtlunhJSqcMOTRFkB0wPQbuNfwSmxzScMc4hQlDYQYUixgSYgLaRinaxyanWIamsu/Vu83DWQ4WOjzeg54DnTJuQVDbgRObK7sPr7zcPEdV4dxNQxDLBJCSVzRGd3+54t7O0K7+6Te6MXZtgxuUwRWGCjpJpL4+8k398OwgCssDRasBi3IpjC6M5DqdnFixBpuSA2bWoAloQzzJVEY1DmpueVOw1NUyKsH9Ph0zWRuKaPNcG1YHkeD1ZylKqnKO2ZAHQjH2qZJ65T/sLYHXVvSrUMlcxTszEUnRYN6WZHg15//rvAapokk919V8XMQL+XJZdNItYao+iwkKL2BdUscXaZSL8O2d6/HFGTdxbJF1GrzA2PpIvEe9NMtdS1vkXfHqfiYSJ7vVRcS0MRAVXnTutCyTMxa/V70rclkTTCYtYqB1gx3K/CNarIAojK1+y4wsRQ7NMj4NjexS+17PLONznZmACwGmEvHAczbJ5XIjOMLYUCjPOYqobUCcAGjnGRiq/NrVYejrBbDE7gDIfFUsndmJ/fIrLZXwAStXc+5Sjd67jY7AJwFAmUM1E7EKJ9sdS2sUGfJ7ToUWpjhUix+HmZu3iosagHq6Fa9Nr93LO9yRn8bABW/yMDsv0o1Abqsz03MqP3ibVcQ73kVkW2ZhTlgY0PQ8SHPhk/oepij0TyLVBnfKsR0tI7brp+266aXnNQZaM4mCtflUlL32hJ6rfTtkSuB7Equhr9UxN3b3fnC6Tk+MbU/XUiQdSAdjZfd5q2HJbUEJZWhUC0yDkuCrbLshSeSt86JmE8NyWcKYhwQ4TvnS7Cu/FgcupQEXi2RmgFi4xKlHql2tUECukhomuG3Hu8OKYLuSFsCWcgolEDT06QOgw5ImGjlReuF1OC3K/QzvlG7dcI0jBh+HKD2+R74xxMJn5LeimMPD7t4vsN2G6qcQyXV3mZ9oZv9VPySSwiwixqwvFHQwyqmiKpogoP1MEo1ew7NYUdp0muQ7OUJtg3fdbUFTgnQXd5MfTJBeQePdiyzZphEsbN0t8aH16ISpcG7c33r2zkMXmpBzXxllXepVtn7Z13IYDt1EwPWouqOMVsakLSeXjKW6FUf2+i27RYQ0ToJG0C+5OhIwFjBMxtc4bqoiVC+D+UINCB1mfcSzQWVdIhYnjN7MhHa78ZWnypGBNHmQy8Zj7zdLcBnJPJnOwVS6dX5jKXDKbnsvkbKjuB9kgGM2TqcwsukycF0OZVGaimgRjnntSmQbcG+fmUrfVnjFZ4JyiVotZFIMHSqazreMzmhUdwQyPcYmdbCsX0k6CfPU1Oa/ANnZozg0MKM/Rpwq9EcjiXBeSyih9UTjZevY06EiVUmPR0WEcLcG29a1CGFDFaMRR28QUIWS2V7SANxKG9iAx8mhY/dMSuR32kKceByu/XarAnmWrhFvuh+05RkCUM277Lsho3Kdd1COtdJ45u4PuIefQ5TLyLQIT3Y9AIaeGDawipjcxTkVCKiCw61oi+G6msMPOMYEzZnpw53sh7Ao+BCncSqsSZKiAM4EhCbfbL21kIalAU6apFHDUbEPClCZhl5mae3r1SRi9VjX3LAd63Zp9lJ96IGpjRhGN+3nm7KpJNdRnGamGEe2TamjSjFR/QCpwRBWTuYZH2y1NwkxJm0FIyBKkNU+n5KtqQN6NfdOofdB8gGLAlBS2SdMe8lzaNGnfq39/i7xrneE86uj//JY94ekGCWCKESEKOZZTsxic2/t5ZPle2b/XvvgCwtZYodDloy+/9BLcYYLLiPJRi5eej+GsY2mP8nqoDabjUUPjfgO2nj4dDRV6N2CBtOo3yTcKfUpjOAVWH5IHY/kj2SN2JTGZr82EJtV/fOA67Qq03YFmZJXaglsQcnMH8t97B3Klqy/rpMhNH3XTR73xPupKSC64qPvPXNH9u4rcXOv9n13rfRXA3FwFfoWrwMsMPXX4IjdHrzdx9Hoj6fDmzvbmzvZKIF11i0tu7nD/F+9w7711v/yv/p8BAAD//1tbErb3HwAA
kind: ConfigMap
metadata:
  creationTimestamp: "2020-01-28T20:16:44Z"
  labels:
    MODIFIED_AT: "1580308975"
    NAME: s3-test
    OWNER: TILLER
    STATUS: SUPERSEDED
    VERSION: "1"
  name: s3-test.v1
  namespace: kube-system
  resourceVersion: "2433822"
  selfLink: /api/v1/namespaces/kube-system/configmaps/s3-test.v1
  uid: 1a4ce385-420b-11ea-aa55-00505697f875
`

I have no idea why the release is like that.

**Comment by amx-jshriner** [comment]
An update...

I manually created a config file with the correct release and namespace. After rebuilding the env from helm with this configmap in the templates folder, I was able to get my OB/OBC launched as Custom Resources in the correct namespace. 

Now, the pod I have instructed to use this claim is now up and green, but I'm having trouble identifying any information in the bucket, or vice-versa, I cannot see the data from the pod in AWS's S3 console for this bucket.

Basically, how can I see files from the bucket on my pod now that it's using it as a PV?
